### PR TITLE
fix(docs) add postgres password

### PR DESCRIPTION
**Pull request by jeveleth**
Postgres has updated all docker images to require password. This change adds the newly required field.

Background: https://github.com/Kong/docs.konghq.com/pull/1794

### CODE REVIEW

**Comment by hutchic** [comment]
could we wedge the CE installation instruction fix onto this PR as well?

File: app/enterprise/0.31-x/installation/docker.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
         -p 5432:5432 \
         -e "POSTGRES_USER=kong" \
         -e "POSTGRES_DB=kong" \
+        -e "POSTGRES_PASSWORD=kong" \
```
[the remark this answers is not in the document]
Comment (jeveleth) [reply app/enterprise/0.31-x/installation/docker.md]: Haven't tested with the `POSTGRES_HOST_AUTH_METHOD` arg, just with `POSTGRES_PASSWORD`.

File: app/enterprise/0.31-x/installation/docker.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
...
+        -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/0.31-x/installation/docker.md]:
```suggestion
        -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

Comment (h3xar0n) [reply app/enterprise/0.31-x/installation/docker.md]: sorry, @jeveleth

File: app/enterprise/0.32-x/installation/docker.md [hunk 3]
```diff
@@ -39,6 +39,8 @@ A guide to installing Kong Enterprise Edition (and its license file) as a Docker
         -p 5432:5432 \
         -e "POSTGRES_USER=kong" \
         -e "POSTGRES_DB=kong" \
+        -e "POSTGRES_PASSWORD=kong" \
+        -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/0.32-x/installation/docker.md]:
```suggestion
        -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

File: app/enterprise/0.33-x/installation/docker.md [hunk 4]
```diff
@@ -56,6 +56,8 @@ You should now have your Kong Enterprise Free Trial image locally. Run
           -p 5432:5432 \
           -e "POSTGRES_USER=kong" \
           -e "POSTGRES_DB=kong" \
+          -e "POSTGRES_PASSWORD=kong" \
+          -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/0.33-x/installation/docker.md]:
```suggestion
          -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

File: app/enterprise/0.34-x/installation/docker.md [hunk 5]
```diff
@@ -53,6 +53,8 @@ is required**:
       -p 5432:5432 \
       -e "POSTGRES_USER=kong" \
       -e "POSTGRES_DB=kong" \
+      -e "POSTGRES_PASSWORD=kong" \
+      -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/0.34-x/installation/docker.md]:
```suggestion
      -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

File: app/enterprise/0.35-x/deployment/installation/docker.md [hunk 6]
```diff
@@ -120,6 +120,8 @@ Kong Enterprise requires a database and supports either Cassandra or PostgreSQL.
         -p 5432:5432 \
         -e "POSTGRES_USER=kong" \
         -e "POSTGRES_DB=kong" \
+        -e "POSTGRES_PASSWORD=kong" \
+        -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/deployment/installation/docker.md]:
```suggestion
        -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

File: app/enterprise/0.36-x/deployment/installation/docker.md [hunk 7]
```diff
@@ -120,6 +120,8 @@ Kong Enterprise requires a database and supports either Cassandra or PostgreSQL.
         -p 5432:5432 \
         -e "POSTGRES_USER=kong" \
         -e "POSTGRES_DB=kong" \
+        -e "POSTGRES_PASSWORD=kong" \
+        -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/0.36-x/deployment/installation/docker.md]:
```suggestion
        -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

File: app/enterprise/1.3-x/deployment/installation/docker.md [hunk 8]
```diff
@@ -64,6 +64,8 @@ $ docker run -d --name kong-ee-database \
   -p 5432:5432 \
   -e "POSTGRES_USER=kong" \
   -e "POSTGRES_DB=kong" \
+  -e "POSTGRES_PASSWORD=kong" \
+  -e "POSTGRES_HOST_AUTH_METHOD: trust" \
```
Comment (h3xar0n) [inline app/enterprise/1.3-x/deployment/installation/docker.md]:
```suggestion
  -e "POSTGRES_HOST_AUTH_METHOD=trust" \
```

**Pull request merged**